src/arrayReverse.test.js

```diff
@@ -16,5 +16,26 @@ describe(`Function 'arrayReverse':`, () => {
     expect(arrayReverse(['Mate', 'Academy'])).toEqual(['ymed', 'acAetaM']);
   });
 
-  // write more tests here
+  it(`should return an array with same length
+  as original array length`, () => {
+    const arrLength = arrayReverse(['Mate', 'Academy']);
+
+    expect(arrLength.length).toBe(2);
+  });
+
+  it(`should not change length of strings in array after reverse`, () => {
+    const arrLength = arrayReverse(['Mate', 'Academy']);
+
+    expect(arrLength[0].length).toBe(4);
+  });
+
+  it(`should return empty array if empty array used as an argument`, () => {
+    const arrLength = arrayReverse([]);
+
+    expect(arrLength).toEqual([]);
+  });
+
+  it(`should not change length of strings in array after reverse`, () => {
+    expect(() => arrayReverse()).toThrow();
```

Comment:
The test description is misleading. It suggests checking the length of strings after reversing, but the test is actually checking if the function throws an error when no arguments are passed. Consider updating the description to accurately reflect the test.